build.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Default">
+    <!-- TODO use common build lifecycle -->
+    <Import Project="build\common.props" />
+
+    <!-- build project dependencies -->
+    <PropertyGroup>
+        <RestoreSources>$(RestoreSources);https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json</RestoreSources>
+        <!-- this build project targets netcoreapp1.0 so it can restore build targets, but doesn't actually compile sources -->
+        <TargetFramework>netcoreapp1.0</TargetFramework>
+    </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="NuGetPackageVerifier" Version="1.0.1-*" />
+    </ItemGroup>
+
+    <!-- build settings -->
+    <PropertyGroup>
+        <ArtifactsPath>$(MSBuildThisFileDirectory)artifacts\build\</ArtifactsPath>
+        <SolutionFile>$(MSBuildThisFileDirectory)DotNetTools.sln</SolutionFile>
+    </PropertyGroup>
+    <ItemGroup>
+        <SourceProjects Include="src\*\*.csproj" />
+        <TestProjects Include="test\*\*.csproj" />
```

Comment:
Would be awesome. My MSBuild-ese isn't quite good enough to know the best way to code a good sorting algorithm like that....does the .NET Core MSBuild support code tasks?